cinema_service/urls.py

```diff
@@ -2,10 +2,49 @@
 from django.conf.urls.static import static
 from django.contrib import admin
 from django.urls import path, include
+from drf_spectacular.views import (
+    SpectacularAPIView,
+    SpectacularRedocView,
+    SpectacularSwaggerView
+)
+from rest_framework_simplejwt.views import (
+    TokenObtainPairView,
+    TokenRefreshView,
+)
 
 urlpatterns = [
     path("admin/", admin.site.urls),
-    path("api/cinema/", include("cinema.urls", namespace="cinema")),
-    path("api/user/", include("user.urls", namespace="user")),
+    path(
+        "api/cinema/",
+        include("cinema.urls", namespace="cinema")
+    ),
+    path(
+        "api/user/",
+        include("user.urls", namespace="user")
+    ),
+    path(
+        "api/doc/",
+        SpectacularAPIView.as_view(), name="schema"
+    ),
+    path(
+        "api/doc/swagger/",
+        SpectacularSwaggerView.as_view(url_name="schema"),
+        name="swagger-ui"
+    ),
+    path(
+        "api/doc/redoc/",
+        SpectacularRedocView.as_view(url_name="schema"),
+        name="redoc"
+    ),
+    path(
+        "api/token/",
+        TokenObtainPairView.as_view(),
+        name="token_obtain_pair"
+    ),
+    path(
+        "api/token/refresh/",
+        TokenRefreshView.as_view(),
+        name="token_refresh"
+    ),
     path("__debug__/", include("debug_toolbar.urls")),
```

Comment:
Ensure that the `debug_toolbar` is only included in development environments. Consider wrapping this path with a condition to check if `settings.DEBUG` is `True` to prevent exposing sensitive information in production.